NetKAN/VenStockRevamp.netkan

```diff
@@ -13,5 +13,15 @@
     "comment"      : ".version file currently has accurate mod version but incorrect KSP version",
 	"resources" : {
         "homepage"     : "http://forum.kerbalspaceprogram.com/threads/92764"
-    }
+    },
+        "x_netkan_override" : [
+        {
+            "version" : "v1.2",
```

Comment:
Oh? I was confused by the example PJF linked, in which the spec_version and "version" were in completely different formats.